operator/v1alpha1/types.go

```diff
@@ -35,6 +35,9 @@ type OperatorSpec struct {
 
 	// logging contains glog parameters for the component pods.  It's always a command line arg for the moment
 	Logging LoggingConfig `json:"logging,omitempty"`
+
+	// revisionHistoryLimit is the number of installer pod attempts to keep a record of (0 being unlimited)
+	RevisionHistoryLimit int `json:"revisionHistoryLimit,omitempty"`
```

Comment:
Only problem I see with this name is that we don't use "revision" in operator. It is common term in workloads/deployment world. I think we call the instances of our deployments "generation". 

Would `DeploymentGenerationHistoryLimit` or `DeploymentHistoryLimit` will be better? 

/cc @openshift/api-review

Reply:
/cc @sttts

Reply:
DeploymentHistoryLimit assumes that the operator is a deployment, or at best will be confusing.

I'm really uncomfortable with the scope of this spec.  Adding this is starting to push it over the edge.

Reply:
OperatorHistoryLimit or OperatorRevisionLimit are probably ok.

Reply:
@smarterclayton this is wrong place to add it, what this does is limit the number of static pod directories we create in static pod installer inside kubelet manifest dir... you set this to 10 and we only keep last 10 static pod manifests + configmaps + secrets... "deployment" here refers to 1 generation of static pod (+all backing resources) on disk.